src/store.rs

```diff
@@ -15,19 +15,59 @@
  */
 
 use murmur::MurmurStore;
+use rocket_db_pools::mongodb::{bson::doc, options::{FindOptions, InsertOneOptions}};
 use std::fs::File;
 
-pub(crate) fn load() -> MurmurStore {
+use rocket_db_pools::Connection;
+use rocket_db_pools::mongodb::Collection;
+use rocket_db_pools::mongodb::bson::oid::ObjectId;
+use rocket::{futures::TryStreamExt, serde::Serialize};
+
+use crate::Db;
+
+pub(crate) fn load_from_file() -> MurmurStore {
 	// TODO: load from DB
 	let mmr_store_file = File::open("mmr_store").expect("Unable to open file");
 	let data: MurmurStore = serde_cbor::from_reader(mmr_store_file).unwrap();
 
 	data
 }
 
+pub(crate) async fn load_from_db(object_id_string: String, db_name: &str, collection_name: &str, db:Connection<Db>, options: Option<FindOptions>) -> MurmurStore {
+
+	let object_id = ObjectId::parse_str(object_id_string).unwrap();
```

Comment:
Let's handle these unwraps, something like this:
- change the return type to `Result<MurmurStore, Error>` where `Error` is an error enum 
- then we can handle the error like
``` rust
let object_id = ObjectId::parse_str(object_id_string)
    .map_err(|_| Error::InvalidObjectIdFormat);
```
same for lines 44 and 46.